File: sdks/swift/Sources/PuxbaySDK/Models/CashDrawerSession.swift
import Foundation

public struct CashDrawerSession: Codable {
    public let id: String
    public let branch: String
    public let openedBy: String
    public let closedBy: String?
    public let openingCash: Double
    public let closingCash: Double?
    public let actualCash: Double?
    public let status: String
    public let openedAt: String
    public let closedAt: String?
}

File: sdks/rust/src/models/cash_drawer_session.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize)]
pub struct CashDrawerSession {
    pub id: String,
    pub branch: String,
    pub opened_by: String,
    pub closed_by: Option<String>,
    pub opening_cash: f64,
    pub closing_cash: Option<f64>,
    pub actual_cash: Option<f64>,
    pub status: String,
    pub opened_at: String,
    pub closed_at: Option<String>,
}
